### PR TITLE
fixed #1384 #1385 #1386 #1387 #1388 #1389

### LINKED ISSUE #1384: Research needed for: shop/convenience|セイコーマート
#### :thinking: We need help deciding what to do with "shop/convenience|セイコーマート"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/convenience|セイコーマート` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/convenience|セイコーマート`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/convenience|セイコーマート`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### PR DESCRIPTION
**Pull request by higa4**
https://github.com/osmlab/name-suggestion-index/issues/1384
https://github.com/osmlab/name-suggestion-index/issues/1385
https://github.com/osmlab/name-suggestion-index/issues/1386
https://github.com/osmlab/name-suggestion-index/issues/1387
https://github.com/osmlab/name-suggestion-index/issues/1388
https://github.com/osmlab/name-suggestion-index/issues/1389

### CODE REVIEW

File: config/canonical.json [hunk 1]
```diff
@@ -23050,12 +23052,14 @@
       "shop": "convenience"
     }
   },
-  "shop/convenience|セーブオン": {
+    "shop/convenience|セーブオン": {
```
Comment (ENT8R) [inline config/canonical.json]: This and all the other spaces look some kind of unintended... Could you maybe remove them?

Comment (matkoniecz) [reply config/canonical.json]: It should be enough to execute `npm run build`.

Comment (good-praxis) [reply config/canonical.json]: Yep, seems like skipping the build. @higa4 please just run the build and push the changes 😁

**Comment by bhousel** [comment]
This was merged in 823ddd2

**Pull request closed without merging**